src/slider/tests/functional/Slider.ts

```diff
@@ -0,0 +1,250 @@
+const { registerSuite } = intern.getInterface('object');
+const { assert } = intern.getPlugin('chai');
+
+import keys from '@theintern/leadfoot/keys';
+import * as css from '../../styles/slider.m.css';
+
+function getPage(test: any) {
+	const { browserName } = test.remote.environmentType;
+	if (browserName.toLowerCase() === 'microsoftedge') {
+		test.skip('example page currently doesn\'t work in edge.');
+	}
+	return test.remote
+		.get('http://localhost:9000/_build/common/example/?module=slider')
+		.setFindTimeout(5000);
+}
+
+function checkValue(command: any, values?: number[]) {
+	const notIE = command.session.capabilities.browserName.toLowerCase() !== 'internet explorer';
+	let currentValue: number;
+	return command
+		.findByCssSelector(`.${css.inputWrapper}`)
+			.findByCssSelector(`.${css.input}`)
+				.getProperty('value')
+				.then((value: string) => {
+					currentValue = parseInt(value, 10);
```

Comment:
I think this works because all the example page sliders are from 0-100, but it's probably worth noting that if that ever changes, this check will fail.